de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUQueryBuilder.java

```diff
@@ -937,9 +999,34 @@ public boolean matches(final IAttributeSet attributes)
 	public IHUQueryBuilder setOnlyWithBarcode(final String barcode)
```

Comment:
please update the javadoc in IHUQueryBuilder to describe what the method is really achieving now